### PR TITLE
Update farmblocks dependencies

### LINKED ISSUE #1059: Update dependencies

### PR DESCRIPTION
**Pull request by alcferreira**
## Closes Issues

* Closes #1059

### Reviewer 1

* [ ] Functions properly
* [ ] Good readability, clarity and overall quality
* [ ] Has tests
* [ ] Has docs

### Reviewer 2

* [ ] Functions properly
* [ ] Good readability, clarity and overall quality
* [ ] Has tests
* [ ] Has docs

### CODE REVIEW

File: packages/input-phone/src/PhoneInput.test.js [hunk 1]
```diff
@@ -94,7 +94,7 @@ describe("PhoneInput", () => {
     test("should search for country", async () => {
       expect(screen.queryByText("+260")).not.toBeInTheDocument();
       // ASCII search
-      userEvent.type(screen.getByTestId("country-search-input"), "Zambia");
+      userEvent.type(screen.getByTestId("country-search-input"), "aibmaz"); // for some reason after upgrading to @testing-library/user-event@13.5.0 the input order has been flipped
```
Comment (ViniciusGularte) [inline packages/input-phone/src/PhoneInput.test.js]: Maybe updating jest don't solve this ?  [Same problem report on stack](https://stackoverflow.com/questions/68164052/why-is-react-testing-library-jest-receiving-dom-input-backwards)

File: packages/icon/buildJSX.mjs [hunk 2]
```diff
@@ -127,6 +124,7 @@ async function buildJSX() {
 
       return Promise.resolve([...allGroupsAcc, ...groupFiles]);
     }, Promise.resolve([]));
+    console.log(">>>>> 3 > files: ", files);
```
Comment (ViniciusGularte) [inline packages/icon/buildJSX.mjs:127]: Please remove the console.log

File: packages/input-phone/src/PhoneInput.test.js (hunk 1 above)
Comment (alcferreira) [reply packages/input-phone/src/PhoneInput.test.js]: Yes, this was resolved after upgrading jest, it's on the last commits

File: packages/icon/buildJSX.mjs (hunk 2 above)
Comment (alcferreira) [reply packages/icon/buildJSX.mjs:127]: oops

Comment (alcferreira) [reply packages/icon/buildJSX.mjs]: fixed

File: packages/input-phone/src/PhoneInput.test.js (hunk 1 above)
Comment (ViniciusGularte) [reply packages/input-phone/src/PhoneInput.test.js]: Nice

**Review by ViniciusGularte — APPROVED** [review]

**Pull request merged**